exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/FilterEvaluatorUtils.java

```diff
@@ -54,9 +54,12 @@
   private FilterEvaluatorUtils() {
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
Please change it to `RedundantTypeArguments`, since current suppression does not help.
```suggestion
  @SuppressWarnings("RedundantTypeArguments")
```